core/storage/question/gae_models.py

```diff
@@ -192,21 +194,37 @@ def create(cls, question_id, skill_id):
         return question_skill_link_model_instance
 
     @classmethod
-    def get_question_ids_linked_to_skill_ids(cls, skill_ids):
+    def get_question_ids_linked_to_skill_ids(cls, skill_ids, start_cursor):
         """Creates a new QuestionSkillLinkModel entry.
 
         Args:
             skill_ids: list(str). The ids of skills for which the linked
                 question ids are to be retrieved.
+            start_cursor: str. The starting point from which the batch of
+                questions are to be returned.
 
         Returns:
-            list(str). The question ids linked to given skills.
+            list(str), str. The question ids linked to given skills and the next
+                cursor value to be used for the next page.
```

Comment:
State that the next cursor value is urlsafe. Ditto for start_cursor -- is it urlsafe? (That's an important thing to mention in the docstring, it defines the interface.)

Reply:
Done.